nightingale_proprioception/CMakeLists.txt

```diff
@@ -0,0 +1,207 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(nightingale_proprioception)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  geometery_msgs
+  rospy
+  std_msgs
+  tf2_ros
+)
+
+## System dependencies are found with CMake's conventions
+# find_package(Boost REQUIRED COMPONENTS system)
+
+
+## Uncomment this if the package has a setup.py. This macro ensures
+## modules and global scripts declared therein get installed
+## See http://ros.org/doc/api/catkin/html/user_guide/setup_dot_py.html
+# catkin_python_setup()
```

Comment:
Might need to uncomment this line

Reply:
I don't have a setup.py in my package since each script is independent of the other, so uncommenting this line leads to a build error